docs/ansible/roles/debops.docker_server/upgrade.rst

```diff
@@ -7,6 +7,13 @@ The upgrade notes only describe necessary changes that you might need to make
 to your setup in order to use a new role release. Refer to the
 changelog for more details about what has changed.
 
+From v1.0.x to master
+---------------------
+
+The role has been renamed from ``debops.docker`` to ``debops.docker_server``.
+The variable prefix has been updated to ``docker_server__``. You will need to
+update your inventory to continue using this role.
```

Comment:
This should be included in the `docs/news/upgrades.rst` file as well. The per-role upgrade files will be removed at some point. I would also mention that the hosts need to be moved to the new Ansible inventory group for the role to be enabled, because it's a very rare change.